index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <link rel="stylesheet" href="style.css">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+
+<body>
+  <section class="intro">
+    <header>
+      <nav>
+        <img src="images/MyLogo.png" alt="Logo">
+        <p>more options</p>
+      </nav>
+    </header>
+    <div class="head">
+      <h1>
+        I’m Lumato <br>
+        Glad to see <br> you!
+      </h1>
+    </div>
+    <div class="content">
+      <p>I’m a software developer! I can help you build a
+        product , feature or website Look through some
+        ce! If you like what you see and have a project you need code
+        d, don’t hestiate to contact me.</p>
+      <p class="connect">LETS CONNECT</p>
+
+      <div class="icons"><img src="images/twitter.png" alt="twitter">
+        <img src="images/in.png" alt="in">
+        <img src="images/M.jpg" alt="m">
+        <img src="images/cat.png" alt="cat">
+        <img src="images/fingers.png" alt="fingers"></div>
+    </div>
```

Comment:
- [ ] Icons should have link `a` tag, be list items with `li` tag and be wrapped in `ul` tag. Kindly make this change. :pray: